### PR TITLE
Make notifications transient if successful

### PR DESCRIPTION
**Pull request by Grafcube**
This PR makes it so that if the exit code of the command is 0, the notification is set as transient.

### CODE REVIEW

File: auto-notify.plugin.zsh [hunk 1]
```diff
@@ -51,10 +51,12 @@ function _auto_notify_message() {
 
     if [[ "$platform" == "Linux" ]]; then
         local urgency="normal"
+        local transient=--transient
         if [[ "$exit_code" != "0" ]]; then
             urgency="critical"
+            transient=""
         fi
-        notify-send "$title" "$body" --app-name=zsh "--urgency=$urgency" "--expire-time=$AUTO_NOTIFY_EXPIRE_TIME"
+        notify-send "$title" "$body" --app-name=zsh $transient "--urgency=$urgency" "--expire-time=$AUTO_NOTIFY_EXPIRE_TIME"
```
Comment (MichaelAquilina) [inline auto-notify.plugin.zsh:59]: please quote the variable here too :)

File: auto-notify.plugin.zsh [hunk 2]
```diff
@@ -51,10 +51,12 @@ function _auto_notify_message() {
 
     if [[ "$platform" == "Linux" ]]; then
         local urgency="normal"
+        local transient=--transient
```
[the remark this answers is not in the document]
Comment (Grafcube) [reply auto-notify.plugin.zsh]: Fixed in 9a9f8f2 :+1:

File: auto-notify.plugin.zsh (hunk 1 above)
Comment (Grafcube) [reply auto-notify.plugin.zsh]: Fixed in 9a9f8f2 :+1:

**Comment by Grafcube** [comment]
Done. I've made the requested changes in 9a9f8f2.

**Pull request merged**